lib/jsdom/living/websocket/WebSocket-impl.js

```diff
@@ -0,0 +1,20 @@
+"use strict";
+const { setupForSimpleEventAccessors } = require("../helpers/create-event-accessor");
+
+const READY_STATES = Object.freeze({
+  CONNECTING: 0,
+  OPEN: 1,
+  CLOSING: 2,
+  CLOSED: 3
+});
+
+const events = ["open", "error", "close"];
+
+class WebSocketImpl {
```

Comment:
This should `extends EventTargetImpl`, after importing `EventTargetImpl` from `require('../events/EventTarget-impl').implementation`.